cargo-afl/build.rs

```diff
@@ -14,67 +13,75 @@ static AR_CMD: &str = "ar";
 mod common;
 
 fn main() {
```

Comment:
This was initially because of the compilation errors we had and discussed before, all these have been reverted to the master branch's way. Done in the latest commit.